frontend/src/components/molecules/NewEnrollmentButton/style.css

```diff
@@ -22,4 +22,5 @@
 
 .new-enrollment-button-item a {
   box-shadow: none;
+  background-image: none;
```

Comment:
you should remove the box-shadow rule and add a `background-color: transparent;` rule like in frontend/src/components/atoms/TagContainer/style.css